docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/ConvertTypeMultiColumn.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    // This example illustrates how to convert multiple columns of different types to one type, in this case System.Single. 
+    // This is often a useful data transformation before concatenting the features together and passing them to a particular estimator.
+    public static class ConvertTypeMultiColumn
+    {
+        // The initial data type
+        private class InputData
+        {
+            public bool Feature1;
+            public string Feature2;
+            public DateTime Feature3;
+            public double Feature4;
+        }
```

Comment:
please move the data classes below example as discussed in the group (i also added this note to the checklist)